src/other/tempo.c

```diff
@@ -41,8 +41,22 @@ void tempo_tick() {
 	// Draw Flashing LEDs
 	u8 flash_state = (tempo_counter % (tempo_bar >> 2)) < (tempo_bar >> 3);
 	for (u8 i = 1; i < 100; i++) {
-		if (flash_screen[i][0] || flash_screen[i][1] || flash_screen[i][2]) {
-			direct_led(i, flash_screen[i][0] * flash_state, flash_screen[i][1] * flash_state, flash_screen[i][2] * flash_state);
+		if (!(flash_screen[i][0] >> 6 || flash_screen[i][1] >> 6 || flash_screen[i][2] >> 6)) {
+			if (flash_state) {
+				direct_led(
+					i,
+					flash_screen[i][0],
+					flash_screen[i][1],
+					flash_screen[i][2]
+				);
+			} else {
```

Comment:
```suggestion
			} else if (flash_screen[i][0] !== led_screen[i][0] || flash_screen[i][1] !== led_screen[i][1] || flash_screen[i][2] !== led_screen[i][2]) {
```


Maybe needed, as if `led_screen`'s color is the same as `flash_screen`'s, it will light the LED for nothing as the color won't change.

WDYT?

Reply:
nah tbh I don't think this matters too much, anyone that's using flashing probably won't be sending rainbow magic light effects that eat a huge amount of bandwidth